byron/ledger/executable-spec/src/Ledger/Update.hs

```diff
@@ -12,6 +13,11 @@
 module Ledger.Update where
 
 import Control.Lens
+import qualified Crypto.Hash as Crypto
+import Data.AbstractSize (HasTypeReps)
+import qualified Data.Bimap as Bimap
```

Comment:
Maybe add `import Data.Bimap (Bimap)` to clean up the type uses

Reply:
Fixed.